### DISCUSSION TITLE
362x used to sell tape?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#135] [General]
If a parity error was found, just one, from the 606 or 607, the controller disconnects, and the channel clear or master clear necessary to reconnect?  With a red light in the unit selector switch at that.  Screw data recovery...

Someone wanted to sell tape.  A crosstown deal?  Maybe off of Minnehaha?

This is just evil.  We are talking about oxides one step above eight track crapola, but guess who had the superior binder of the day...

**Comment by KC5VDJ** [comment]
My bad.  More like whomever wrote and whomever reviewed the manual not elaborating where elaboration seems needed.  Seems there are commands and such to deal with this.

I shouldn't have to read several sections to learn the commands to deal with such bitrot / line garbage, after a statement making it seem that such might not exist.  Where is that DeLorian?  Send me to 1965 to ask why...  Oh wait, not possible, it was destroyed behind Surfside 3 right off Ventura Road...  Damn.

**Comment by wschaub** [comment]
There are two kinds of parity error on the 362x controller, the one I think you're reading about is for parity errors detected over the channel vs detected on tape. for tape parity errors they are flagged in the status register plus the 6681 interrupt status bits and there are clearing codes for that, the parity error flag also gets reset on the next code that causes tape motion that moves the tape away from the section that had the parity error. 

There's a lot of weird non-obvious things that go on with 7 track tape that I'm quickly finding out.  having both odd and even parity possible on the same tape really complicates things.
